package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
Why was this added? This has nothing to do with the translation and should not be part of this commit.